hack/ci/windowsRS1.ps1

```diff
@@ -207,8 +207,8 @@ Try {
     $origGOPATH="$env:GOPATH"        # So we can restore it at the end
 
     # Turn off progress bars
-	$origProgressPreference=$global:ProgressPreference
-	$global:ProgressPreference='SilentlyContinue'
+    $origProgressPreference=$global:ProgressPreference
+    $global:ProgressPreference='SilentlyContinue'
```

Comment:
Perhaps this commit (cleanup whitespace) should be squashed with the commit that adds the file